.github/workflows/docs-ci-v2.yml

```diff
@@ -7,21 +7,37 @@ on:
     branches: [ 'main', 'release/v*' ]
 
 jobs:
+  makedirs:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Make Directories
+        if: ${{ github.event_name == 'push' }}
```

Comment:
I don't think it makes sense to conditionally run this - might as well run it every time? Otherwise, looks good.